recipes/igblast/meta.yaml

```diff
@@ -24,7 +24,7 @@ requirements:
   run:
     - perl
     - libgcc  # [linux]
-    - zlib
+    - zlib {{CONDA_ZLIB}}*
```

Comment:
pinning on run-only dep can be removed (needs https://github.com/bioconda/bioconda-utils/pull/264 to make linter happy)

Reply:
Same here, I think this can only improve our packages.

Reply:
This is a different case than the above. Here we have a run-only requirement, so the built package should be the same whether you use one or another version for `CONDA_ZLIB`. So it doesn't really make sense to rebuild the package when `CONDA_ZLIB` is updated since it should be identical to the previous build. IMO, run-only pins should never depend on values of `CONDA_*` variables. If the packages really need version constraints, then those have to be given explicitly, e.g., `zlib >=1.2,<1.3.0a` and such.